hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/store/impl/TestSQLFederationStateStore.java

```diff
@@ -76,38 +107,479 @@ public void testSqlConnectionsCreatedCount() throws YarnException {
         FederationStateStoreClientMetrics.getNumConnections());
   }
 
-  @Test(expected = NotImplementedException.class)
   public void testAddReservationHomeSubCluster() throws Exception {
```

Comment:
Should we just remove this?
It will call the parent methods by default right?
We should check the unit test report.

Reply:
> Should we just remove this?
> It will call the parent methods by default right?

Your understanding is accurate, we can delete all this method. When the test is executed, it will call the parent methods by default, but deleting it directly will bring some difficulties to the review, because it will be difficult to compare the code. I will later Submit a refactored pr. For code refactoring, these redundant codes will be deleted directly at that time.

Reply:
Can we do this in this PR?
We can check the unit tests results for all this.

Reply:
I will modify the code, but when the code is compared, there may be misalignments.